rest/util.py

```diff
@@ -60,3 +61,19 @@ def dispatch(self, request, *args, **kwargs):
             CheckValidMemberParamMixin,
             self
         ).dispatch(request, *args, **kwargs)
+
+
+def default_slugify(label, default_name, exists_func):
```

Comment:
I'm not sure on the usefulness of `default_name` here. Why not just use 'slug' as a default term no matter where the slug is assigned? No strong feelings however